src/ida_lunde_naalsund_ex/ex01/tidy_code.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+
+__author__ = 'Ida Lunde Naalsund'
+__email__ = 'idna@nmbu.no'
+
+
+from random import randint as rdint
+
+
+def make_user_guess():
+    """Makes user guess a number.
+    :return: Guess made by user
+    """
+    user_guess = 0
+    while user_guess < 1:
+        user_guess = int(input('Your guess: '))
+    return user_guess
+
+
+def dice_throw():
+    """Simulates two dice throws.
+    :return: Random number between 2 and 12
+    """
+    return rdint(1, 6) + rdint(1, 6)
+
+
+def compare_throw_and_guess(throw, guessed_number):
```

Comment:
could be named `is_equal()` or something. Very simple functions like this should be understood without their context. Another alternative is to remove the function entirely and do the comparison directly in the main section.